### PR TITLE
update group-check

### PR DESCRIPTION
**Pull request by Borda**
## What does this PR do?

Fixes https://github.com/Lightning-AI/lightning/pull/17502#discussion_r1209398725

<details>
  <summary><b>Before submitting</b></summary>

- Was this **discussed/agreed** via a GitHub issue? (not for typos and docs)
- [x] Did you read the [contributor guideline](https://github.com/Lightning-AI/lightning/blob/master/.github/CONTRIBUTING.md), **Pull Request** section?
- [x] Did you make sure your **PR does only one thing**, instead of bundling different changes together?
- Did you make sure to **update the documentation** with your changes? (if necessary)
- Did you write any **new necessary tests**? (not for typos and docs)
- [x] Did you verify new and **existing tests pass** locally with your changes?
- Did you list all the **breaking changes** introduced by this pull request?
- Did you **update the CHANGELOG**? (not for typos, docs, test updates, or minor internal changes/refactors)

</details>

## PR review

Anyone in the community is welcome to review the PR.
Before you start reviewing, make sure you have read the [review guidelines](https://github.com/Lightning-AI/lightning/wiki/Review-guidelines). In short, see the following bullet-list:

<details>
  <summary>Reviewer checklist</summary>

- [x] Is this pull request ready for review? (if not, please submit in draft mode)
- [x] Check that all items from **Before submitting** are resolved
- [x] Make sure the title is self-explanatory and the description concisely explains the PR
- [x] Add labels and milestones (and optionally projects) to the PR so it can be classified

</details>

### CODE REVIEW

File: tests/parity_fabric/run_standalone_tasks.sh [hunk 1]
```diff
@@ -39,5 +39,4 @@ retry_command() {
   return $exit_code
 }
 
-retry_command "python -m test_parity_ddp --accelerator="cpu" --devices=2 --tolerance=0.02"
```
Comment (awaelchli) [inline tests/parity_fabric/run_standalone_tasks.sh:42]: This is a test, it has to stay. Please don't remove this.

**Review by awaelchli — CHANGES_REQUESTED** [review]

File: tests/parity_fabric/run_standalone_tasks.sh (hunk 1 above)
Comment (Borda) [reply tests/parity_fabric/run_standalone_tasks.sh:42]: That was asked by @carmocca, but happy to keep it...

File: tests/parity_fabric/run_standalone_tasks.sh [hunk 2]
```diff
@@ -39,5 +39,4 @@ retry_command() {
...
 retry_command "python -m test_parity_ddp --accelerator="cuda" --devices=2 --tolerance=0.01"
```
Comment (Borda) [inline tests/parity_fabric/run_standalone_tasks.sh:42]:
```suggestion
retry_command "python -m test_parity_ddp --accelerator="cpu" --devices=2 --tolerance=0.02"
retry_command "python -m test_parity_ddp --accelerator="cuda" --devices=2 --tolerance=0.01"
```

File: tests/parity_fabric/run_standalone_tasks.sh (hunk 1 above)
Comment (carmocca) [reply tests/parity_fabric/run_standalone_tasks.sh]: Where have I asked this?

Comment (Borda) [reply tests/parity_fabric/run_standalone_tasks.sh]: https://github.com/Lightning-AI/lightning/pull/17502#discussion_r1209399076

Comment (Borda) [reply tests/parity_fabric/run_standalone_tasks.sh]: ok, I was confused with the comment location...

**Review by awaelchli — APPROVED** [review]

**Review by carmocca — APPROVED** [review]

**Pull request merged**